babel/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "example-applications-babel",
+  "version": "1.0.0",
+  "description": "Babel example",
+  "scripts": {
+    "test": "mocha"
+  },
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "directories": {
+    "lib": "./src",
+    "test": "./test"
+  },
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.3.4",
+    "@babel/preset-env": "^7.3.4",
+    "@babel/register": "^7.0.0"
+  },
+  "devDependencies": {
+    "mocha": "^6.0.2"
+  }
+}
```

Comment:
Can we order the keys in _all_ example "package.json" files as follows?
1. "name"
1. "version"
1. "description"
1. "main"
1. "scripts"
1. "directories"
1. "dependencies"
1. "devDependencies"
1. "peerDependencies"
1. "engines"
1. "keywords"
1. "author"
1. "license"